frontend-react/src/components/MessageTracker/WarningsErrors.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+
+// TODO: move this interface into the resources directory
+interface Warnings {
+    field: string | undefined;
+    description: string | undefined;
+    type: string | undefined;
+    trackingIds: string[] | undefined;
+}
+
+type WarningsErrorsDisplayProps = {
+    title: string;
+    data: Warnings[];
+};
+
+export const WarningsErrors = ({ title, data }: WarningsErrorsDisplayProps) => {
+    return (
+        <>
+            <section className="margin-bottom-5">
+                <h3>{title}</h3>
+                <table
```

Comment:
Is this something we could use the `Table` component for? The [DeliveryFacilitiesTable](https://github.com/CDCgov/prime-reportstream/blob/master/frontend-react/src/pages/deliveries/details/DeliveryFacilitiesTable.tsx) is a good example of a simpler table config.

Reply:
I thought about it and didn't implement it for reasons. Will discuss with you and Aurora

Reply:
Sounds good! 👍🏼